Okta.AspNet.WebApi.IntegrationTest/MiddlewareShould.cs

```diff
@@ -66,6 +66,20 @@ public async Task Returns401WhenAccessToProtectedRouteWithInvalidTokenAsync()
             }
         }
 
+        [Fact]
+        public async Task InvokeCustomEventsAsync()
+        {
+            var accessToken = "thisIsAnInvalidToken";
+            HttpRequestMessage request = new HttpRequestMessage(HttpMethod.Get, ProtectedEndpoint);
+            request.Headers.Authorization = new AuthenticationHeaderValue("Bearer", accessToken);
+
+            using (var client = new HttpClient(_server.Handler))
+            {
+                var response = await client.SendAsync(request);
+                Assert.True(response.Headers.Contains("myCustomHeader"));
```

Comment:
Where is "myCustomHeader" defined?

Reply:
I think it's here, line 27 https://github.com/okta/okta-aspnet/blob/2f053ab6133f901b0ff61d48d970f044b29d7781/Okta.AspNet.WebApi.IntegrationTest/Startup.cs#L20